plenigo_wordpress.php

```diff
@@ -39,8 +39,12 @@
 
 // Plenigo JavaScript SDK / Services
 define('PLENIGO_SVC_URL', "https://api.plenigo.com");
+define('PLENIGO_OAUTH_SVC_URL', "https://www.plenigo.com");
 define('PLENIGO_JSSDK_URL', "https://static.plenigo.com");
 
+// Default URL for Javascript
+define('DEF_PLENIGO_SVC_URL', "https://api.plenigo.com");
```

Comment:
This is for comparing and detecting If I changed the JavaScript SDk origin to our test system and automatically put that into plenigo.baseURI